Makefile

```diff
@@ -78,8 +79,26 @@ debug:
 strict:
 	$(eval CFLAGS=-Wall -Werror -Wextra)
 
+clang_find_files_to_lint:
+	@find ./ -type f \
+	-path ./tmp -prune -o \
+	-path ./vendor -prune -o \
+	-path ./tests -prune -o \
+	-path ./src/ext/mpack -prune -o \
+	-iname "*.h" -o -iname "*.c"
+
+clang_format_check:
```

Comment:
❤️

Reply:
Thanks for implementing it, the check internally should produce more readable output too